### DISCUSSION TITLE
How to fill Object Page Header title and Header Facet forms using Test Integration Library

### DISCUSSION THREAD

**Discussion by vaibhav0461** [ui5-community/wdi5#596] [Q&A]
We are new in using wdi5 to write e2e tests for UI Screens which uses Fiori Elements and odata v4. we are creating new Business Object so we are able to click on create button in list report and navigate to Object Page now we want to enter values in Header title, description and in Header Facets. We read through the documentation of sap.fe.test library but not able to make it success.  This is the code we tried.

![Screenshot 2024-03-16 at 3 47 23 PM](https://github.com/ui5-community/wdi5/assets/46601979/b385c9c8-ee57-435b-8947-06b4d5bfb788)
Screen looks like this
![Screenshot 2024-03-16 at 3 47 52 PM](https://github.com/ui5-community/wdi5/assets/46601979/fe728530-e8a8-43a0-a452-9c26607c4fbc)

This is the annotation file for the Object Page Header section
![Screenshot 2024-03-16 at 3 44 21 PM](https://github.com/ui5-community/wdi5/assets/46601979/4110d1cc-f2fd-4e40-ab56-39513313e3fc)

**Comment by Siolto** [comment]
Hi @vaibhav0461,

Try something like this:

```javascript
When.onClassObjectPage.onForm("General Data")
    .iChangeField("Brief Description", "something")
```
